dataverse-persistence/src/test/resources/dbinit.sql

```diff
@@ -95,8 +98,9 @@ INSERT INTO dvobject (id, dtype, owner_id, previewimageavailable, protocol, auth
                       creator_id, createdate, modificationtime, permissionmodificationtime, indextime, permissionindextime)
     VALUES (66, 'Dataset', 1, false, 'doi', '10.18150', 'FK2/QTVQKL', NULL, false, 'file://10.18150/FK2/QTVQKL', NULL, NULL,
             1, '2019-09-26 11:43:58.194', '2019-09-26 11:43:58.194', '2019-09-26 11:43:58.194', NULL, NULL);
+
 INSERT INTO dataset (id, fileaccessrequest, harvestidentifier, usegenericthumbnail, citationdatedatasetfieldtype_id, harvestingclient_id, guestbook_id, thumbnailfile_id)
-    VALUES (66, NULL, NULL, false, NULL, NULL, NULL, NULL);
+    VALUES (66, NULL, NULL, false, NULL, 12, NULL, NULL);
```

Comment:
I know that it is kind of cumbersome, but I would add extra dataset that is harvested and not overwrite existing one here. Reason behind why I think that it is important is: draft datasets often have some different permissions checking. And there are strict limitations to what can be done with harvested datasets. So I think these two kinds of datasets should coexsist in test database.